.github/actions/cdn_code_deploy/action.yaml

```diff
@@ -0,0 +1,100 @@
+name: CDN Code Deploy 
+description: An action Deploy the artifact to the CDN
+
+inputs:
+  inputs:
+    resource_group_name:
+      description: CDN resource group name.
+      type: string
+      required: true
+    storage_account_name:
+      description: Storage account name.
+      type: string
+      required: true
+    profile_cdn_name:
+      description: CDN Profile name.
+      type: string
+      required: true
+    endpoint_name:
+      description: CDN endpoint name.
+      type: string
+      required: true
+    blob_container_name:
+      description: Blob container name.
+      type: string
+      required: true
+    content_paths:
+      description: If you want to include only specific file set 'true' else 'false'
```

Comment:
What do you mean with "specific file" here?

Reply:
You can decide to purge the entire folder or only the files that have been added and/or modified